htdocs/fourn/class/fournisseur.facture.class.php

```diff
@@ -214,6 +214,16 @@ class FactureFournisseur extends CommonInvoice
      */
     public $fk_facture_source;
 
+    /*
+    * BACKPORT - 10/01/2022
+    * Template supplier invoces
+    */
+    public $fac_rec;
```

Comment:
Il manque la définition de l'attribut `fk_fac_rec_source` puisque tu l'as ajouté dans les fichiers SQL
```suggestion
    public $fac_rec;

    /** @var int $fk_fac_rec_source */
    public $fk_fac_rec_source;
```

Reply:
Done